apps/chat/src/utils/server/server.ts

```diff
@@ -26,9 +26,11 @@ export class ServerUtils {
       const text = await res.text();
       try {
         const json = JSON.parse(text);
-        return typeof json === 'string' ? json : JSON.stringify(json);
+        return decodeURIComponent(
+          typeof json === 'string' ? json : JSON.stringify(json),
+        );
```

Comment:
Maybe ?
```suggestion
        return typeof json === 'string' ? decodeURIComponent(json) : JSON.stringify(json);
```

Reply:
In case when json is parsed into js object result can be returned with special symbols, so I think it whould be better to wrap all of this by decoder, or do not wrap json result at all